portal-survey-2023.Rmd

```diff
@@ -0,0 +1,261 @@
+---
+title: "NF-OSI Data Access Survey -- 2023 Preliminary Results"
+description: |
+  Preliminary survey findings from the NF community on the experience of data discovery, sharing, and reuse through the NF Data Portal.
+author:
+  - name: Anh Nguyet Vu
+    url: https://github.com/anngvu
+    affiliation: Sage Bionetworks
+    affiliation_url: https://sagebionetworks.org/
+    orcid_id: 0000-0003-1488-6730
+  - name: Jineta Banerjee
+    url: https://github.com/jaybee84
+    affiliation: Sage Bionetworks
+    affiliation_url: https://sagebionetworks.org/
+    orcid_id: 0000-0002-1775-3645
+  - name: Christina Conrad
+    url: https://github.com/cconrad8
+    affiliation: Sage Bionetworks
+    affiliation_url: https://sagebionetworks.org/
+    orcid_id: 0000-0001-8688-2523
+  - name: Robert Allaway
+    url: https://github.com/allaway
+    affiliation: Sage Bionetworks
+    affiliation_url: https://sagebionetworks.org/
+    orcid_id: 0000-0003-3573-3565
+date: "`r Sys.Date()`"
+output: distill::distill_article
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+library(gt)
+library(googlesheets4)
+library(tidygraph)
+library(ggraph)
+
+source("R/circlepack.R")
+```
+
+
+```{r, include=FALSE}
+sheet_id <- "1mZ6_XzC1d82cCl8uBqBjSDvIMo7-QN4Q2_FMIifbmkA"
+raw <- googlesheets4::read_sheet(ss = sheet_id)
+
+```
+
+```{r, echo=F}
+# users
+emails <- raw %>%
+  pull(`Email Address`) %>%
+  unique()
+
+deid <- setNames(1:length(emails), emails)
+survey <- raw %>%
+  mutate(id = deid[`Email Address`]) %>%
+  select(-c(`Email Address`))
+
+# use only most recent response where some users submitted multiple responses
+survey <- survey %>%
+   group_by(id)  %>%
+   top_n(n = 1, Timestamp)
+```
+
+## Introduction
+
+Since 2015, the NF-OSI data platform has continued to grow and is now the entrypoint to over a hundred research studies for the Neurofibromatosis (NF) research community.
+Eight years later, with the support of the Neurofibromatosis Therapeutic Acceleration Program (NTAP), we surveyed NF community members on the experience of **data findability, sharing, and reuse** through the NF Data Portal. The survey aimed evaluate pain points or barriers in these three key areas to understand where the most impactful improvements could be made.
+
+## Results and Analysis
+
+### Respondent profiles
+```{r fig-respondent-type, fig.cap="Demographic representation of survey respondents", layout="l-body-outset", fig.width=9, fig.height=7, echo=F}
+
+# Format labels
+survey$`What is your professional title?` <- gsub("Bioinformatician/computational biologist", "Bioinformatician\n/ comp. biologist", survey$`What is your professional title?`)
+survey$`What is your professional title?` <- gsub("Bench researcher/staff scientist", "Bench researcher\n/ staff scientist", survey$`What is your professional title?`)
+type_representation(`What is your professional title?`, survey)
+
+```
```

Comment:
I would leave the more comment versions on sample size limitations to the end, and just state it as a fact "Here we surveyed 13 members of the NF community"